{
  super(message,cause);
  setCausedByNetworkIssue(cause instanceof IOException);
}
